### DISCUSSION TITLE
Blink UX

### DISCUSSION THREAD

**Discussion by faermanj** [CaravanaCloud/blink#13] [Ideas]
Hello friends :)

This thread is to define how the system is going to work. Meaning, what is the first step we're going to try to increase awareness regarding blood donation and get more people donating blood and other vitals, like bone marrow or even hair. 

Here are a few initial ideas, please post yours freely so that we can elaborate upon them.

- A gamified system, where users are incentivized to come back frequently.
- Let organizations (schools, companies, etc) create donation campaigns.
- Let people donate money to campaigns, and build competition across campaigns.
- Account support in social media for campaigns.
- Have a chatbot to answer FAQ
- UI: Game over Map (Pokemon Go, Ingress, ...)

What do you think?

**Comment by carlosgc** [comment]
- Let donation centers create alerts when theirs blood supplies are low

**Reply by faermanj** [reply]
Also let them update profile information, frequent questions and so on
